### PR TITLE
Calling Keychecker before checking key in MessageMap

### PR DESCRIPTION
**Pull request by cyyber**
This is fix regarding the issue #4029

### CODE REVIEW

**Comment by cyyber** [comment]
I signed it!

**Comment by acozzette** [comment]
ok to test (this comment will start the other CI runs)

File: python/google/protobuf/internal/message_test.py [hunk 1]
```diff
@@ -1481,11 +1481,8 @@ def testMapGet(self):
     submsg = msg.map_int32_foreign_message[5]
     self.assertIs(submsg, msg.map_int32_foreign_message.get(5))
     # TODO(jieluo): Fix python and cpp extension diff.
```
Comment (anandolee) [inline python/google/protobuf/internal/message_test.py]: Please remove the TODO as well

**Comment by anandolee** [comment]
The change is OK to merge after 'remove the TODO' and jenkins test.

File: python/google/protobuf/internal/message_test.py (hunk 1 above)
Comment (cyyber) [reply python/google/protobuf/internal/message_test.py]: Ok

Comment (cyyber) [reply python/google/protobuf/internal/message_test.py]: TODO has been removed.

**Pull request merged**